Fix implicit declaration of functions.
Has been reported to the developers by email.
--- chklastlog.c.orig	2021-03-07 22:40:27.000000000 -0600
+++ chklastlog.c	2022-03-22 14:14:07.000000000 -0500
@@ -37,9 +37,7 @@
 int main () { return 0; }
 #else
 #include <stdio.h>
-#ifdef __linux__
 #include <stdlib.h>
-#endif
 #include <sys/stat.h>
 #include <unistd.h>
 #include <string.h>
@@ -60,7 +58,6 @@
 #define LASTLOG_FILENAME "/var/log/lastlog"
 #endif
 #ifdef __OpenBSD__
-#include <stdlib.h> 
 #define WTMP_FILENAME "/var/log/wtmp"
 #define LASTLOG_FILENAME "/var/log/lastlog"
 #endif
